packages/core/types/src/modules/entity-service/params/attributes/mocks.ts

```diff
@@ -0,0 +1,13 @@
+import type { Attribute } from '../../../../types';
```

Comment:
Minor thing, maybe it's me not used to using "mocks" in this case, is it mocking types?
I rather see them as primitives, but unsure if they are used for something else

Reply:
Thanks for pointing that out, I wasn't sure about the naming.

We're "overriding" types' value for the original values.

e.g. boolean is not just boolean anymore, it's boolean | 1 | 0 | '1' | '0' | 'false' | 'true', etc...

Hence the "literal value mocks" (or overrides?)

Do you have a better naming idea by any chance?

Reply:
Ah 💡 I see, then `mock`makes sense in this case! thank you

Reply:
I also found mocks to be confusing. It's more like API contract values, right? Or maybe just going with "literals" makes more sense than "mocks"?

Reply:
I can switch to literals to avoid confusion

Reply:
done in [01b5382](https://github.com/strapi/strapi/pull/19011/commits/01b53828d62ea4bb360aafdade04be9b8617fb16)